Games/GrayscaleTest/GrayscaleTest.py

```diff
@@ -4,20 +4,21 @@
 # Shows six different screens, then reboots. Cycle through the screens by
 # pressing A or B.
 
-# Fix import path so it finds the grayscale library
-import sys
-sys.path.insert(0, "/".join(__file__.split("/")[0:-1]))
+# Fix import path so it can find a local grayscale library if present
+if '__file__' in globals():
+    from sys import path as syspath
+    syspath.insert(0, "/".join(__file__.split("/")[0:-1]))
```

Comment:
If this file is `Games/GrayscaleTest/GrayscaleTest.py`, why not just do `syspath.insert('/Games/GrayscaleTest')`? That should work for Fast Execute as well as it doesn't need a `__file__` value, right?